Tests fail with Node.js >= 12

Hi,
The test fail with Node.js >= 12 with the following log: 
```
[master][~/ups/to-fast-properties]$ npm run test       
(node:21169) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

> to-fast-properties@3.0.0 test /home/nilesh/ups/to-fast-properties
> node --allow-natives-syntax test.js && tsd

  ✖ main   object has fast properties

  ✖ 1 test failed

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! to-fast-properties@3.0.0 test: `node --allow-natives-syntax test.js && tsd`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the to-fast-properties@3.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

seems to fail at this step:

```
t.assert(%HasFastProperties(object), 'object has fast properties');
```

--- a/index.js
+++ b/index.js
@@ -21,14 +21,13 @@ function FastObject(o) {
 	return new FastObject;
 }
 
-const kInlineCacheCutoff = 10;
+const inlineCacheCutoff = 10;
 
-// Initialize the inline property cache of FastObject
-for(let i = 0; i <= kInlineCacheCutoff; i++) {
+// Initialize the inline property cache of FastObject.
+for (let i = 0; i <= inlineCacheCutoff; i++) {
 	FastObject();
 }
 
 module.exports = function toFastproperties(o) {
 	return FastObject(o);
 };
-

--- a/index.js
+++ b/index.js
@@ -21,9 +21,14 @@ function FastObject(o) {
 	return new FastObject;
 }
 
+const kInlineCacheCutoff = 10;
+
 // Initialize the inline property cache of FastObject
-FastObject();
+for(let i = 0; i <= kInlineCacheCutoff; i++) {
+	FastObject();
+}
 
 module.exports = function toFastproperties(o) {
 	return FastObject(o);
 };
+

